pertpy/tools/_metadata/_look_up.py

```diff
@@ -340,3 +376,63 @@ def available_genes_annotation(
             print(f"{len(not_matched_identifiers)} genes are not found in the metadata.")
             print(f"{identifier_num_all - len(not_matched_identifiers)} genes are found! ")
         """
+
+    def available_moa(
+        self,
+        query_id_list: list[str] | None = None,
+        target_list: list[str] | None = None,
+    ) -> None:
+        """A brief summary of MoA annotation.
+
+        Args:
+            query_id_list: A list of unique perturbagens to test the number of matched ones present in the metadata. Defaults to None.
+            target_list: A list of unique molecular targets to test the number of matched ones present in the metadata. Defaults to None.
+
+        """
+        if self.type != "moa":
+            raise ValueError("This is not a LookUp object specific for MoaMetaData!")
+        if query_id_list is not None:
+            identifier_num_all = len(query_id_list)
+            not_matched_identifiers = list(set(query_id_list) - set(self.moa_meta.pert_iname))
+            print(f"{len(not_matched_identifiers)} perturbagens are not found in the metadata.")
+            print(f"{identifier_num_all - len(not_matched_identifiers)} perturbagens are found! ")
+
+        if target_list is not None:
+            targets = self.moa_meta.target.astype(str).apply(lambda x: x.split("|"))
+            all_targets = [t for tl in targets for t in tl]
+            identifier_num_all = len(target_list)
+            not_matched_identifiers = list(set(target_list) - set(all_targets))
+            print(f"{len(not_matched_identifiers)} molecular targets are not found in the metadata.")
+            print(f"{identifier_num_all - len(not_matched_identifiers)} molecular targets are found! ")
+
+
+def available_compound(
```

Comment:
```suggestion
def available_compounds(
```

Plural reads nicer I think?

Reply:
We should then use the plural across all functions where it reads better

Reply:
currently only two are in plural (compound + cell line). what about bulk RNA expression, protein expression and moa? i keep it as singular first